### ISSUE TITLE
Improve py4py documentation

### ISSUE THREAD

**Issue by smangham** [sirocco-rt/sirocco#809]
A few small issues with the py4py reverb docs:
* The `continuum` argument for TF creation doesn't specify it should be central source + disk luminosity.
* The `keplerian` argument for TF.plot doesn't explain it should be in r_ISCO.

**Comment by smangham** [comment]
The `threshold` argument for `TransferFunction.delay` doesn't mention it's fractional. It should also mention .8 was the 'standard', ideally referencing the paper I got that from (which I can't remember right now!).

**Comment by smangham** [comment]
`TransferFunction.plot` is designed to chain and returns the `TransferFunction` on completion. It should have an option to return the actual figure created, in case users want to modify it further.

**Comment by smangham** [comment]
I forgot I still had a large script called "py4py,.py" in there that I was in the process of breaking up into smaller ones. I'll check this. Also, at some point (???) I decided to setup [smangham/py4py](https://github.com/smangham/py4py) separately from this repo. Presumably I was going to maintain it as its own thing then gave up and folded it in. I'll delete that repo.

**Comment by smangham** [comment]
I forgot I still had a large script called "py4py,.py" in there that I was in the process of breaking up into smaller ones. I'll check this. Also, at some point (???) I decided to setup [smangham/py4py](https://github.com/smangham/py4py) separately from this repo. Presumably I was going to maintain it as its own thing then gave up and folded it in. I'll delete that repo.

During this, I've discovered that the autodoc isn't parsing the Google-format docstrings properly - it's reading the arguments as block text.
